File: CS/MyWinExplorerView/MyTileViewInfoRegistrator.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using DevExpress.XtraGrid;
using DevExpress.XtraGrid.Registrator;
using DevExpress.XtraGrid.Views.Base;
using DevExpress.XtraGrid.Views.Base.Handler;
using DevExpress.XtraGrid.Views.Base.ViewInfo;

namespace MyXtraGrid
{
    public class MyTileViewInfoRegistrator : TileViewInfoRegistrator
    {
        public override string ViewName { get { return "MyTileView"; } }
        public override BaseView CreateView(GridControl grid) { return new MyTileView(); }
        public override BaseViewInfo CreateViewInfo(BaseView view) { return new MyTileViewInfo(view as MyTileView); }
    }
}

File: VB/MyWinExplorerView/MyTileViewInfoRegistrator.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports System.Threading.Tasks
Imports DevExpress.XtraGrid
Imports DevExpress.XtraGrid.Registrator
Imports DevExpress.XtraGrid.Views.Base
Imports DevExpress.XtraGrid.Views.Base.Handler
Imports DevExpress.XtraGrid.Views.Base.ViewInfo

Namespace MyXtraGrid
    Public Class MyTileViewInfoRegistrator
        Inherits TileViewInfoRegistrator

        Public Overrides ReadOnly Property ViewName() As String
            Get
                Return "MyTileView"
            End Get
        End Property
        Public Overrides Function CreateView(ByVal grid As GridControl) As BaseView
            Return New MyTileView()
        End Function
        Public Overrides Function CreateViewInfo(ByVal view As BaseView) As BaseViewInfo
            Return New MyTileViewInfo(TryCast(view, MyTileView))
        End Function
    End Class
End Namespace
